src/ipc-handlers.ts

```diff
@@ -1328,6 +1330,192 @@ export async function trustCertificate( event: IpcMainInvokeEvent ): Promise< vo
 	}
 }
 
+export function showSiteContextMenu(
+	event: IpcMainInvokeEvent,
+	siteId: string,
+	siteName: string,
+	sitePath: string,
+	isRunning: boolean,
+	isLoading: boolean,
+	isAddingSite: boolean,
+	finderLabel: string,
+	editorLabel: string | null,
+	terminalLabel: string
```

Comment:
Let's remove the unused parameters, `siteName` and `sitePath`, since they are not used. We'll need to update preload.ts and also the usage in site-menu.tsx.

We could also consider wrapping these parameters in a single object called siteInfo or something similar, but it’s not a strong preference.

```suggestion
	siteId: string,
	isRunning: boolean,
	isLoading: boolean,
	isAddingSite: boolean,
	finderLabel: string,
	editorLabel: string | null,
	terminalLabel: string
```

Reply:
Done!
https://github.com/Automattic/studio/pull/1796/commits/846868fd21ef1f09730543072963e012e5394654
https://github.com/Automattic/studio/pull/1796/commits/ab5ac3ff6f85831de456d7b31c7ec0fe4fa65ae5